### PR TITLE
AB#10781

### PR DESCRIPTION
**Pull request by MartinGbz**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (refactor or addition to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/safe/src/lib/style/survey.scss [hunk 1]
```diff
@@ -10,3 +10,19 @@
...
+}
+
+::ng-deep .sv_main .sv_q_erbox:not([style*="display: none"]):not([style*="display:none"]){
+  background-color: transparent;
+  border-width: 0;
+  font-weight: bold;
+  order: 2;
+}
+
+.sv_qstn .content-left {
```
Comment (AntoineRelief) [inline projects/safe/src/lib/style/survey.scss]: @MartinGbz we should not apply the css to this specific question configuration

**Comment by AntoineRelief** [comment]
@MartinGbz this looks almost good.
I think we're just missing how we determine the css of the question, to have its content flexible, and use the order property

I found this link: https://surveyjs.io/Examples/Library?id=survey-cssclasses#content-js

if we look at the code it seems possible to have a custom cssContent property, just need to determine how to set that

Capture d’écran 2021-09-27 à 16 57 19

File: projects/safe/src/lib/style/survey.scss [hunk 2]
```diff
@@ -10,3 +10,19 @@
 ::ng-deep .sv_q_required_text {
     color: red;
 }
+
+::ng-deep .sv_main .sv_p_root .sv_row .sv_q.sv_qstn:first-child:last-child > .content-left {
```
Comment (AntoineRelief) [inline projects/safe/src/lib/style/survey.scss]: @MartinGbz ⚠️ I will correct that, but as explained previously the .content-left class is too specific, and cannot be used. It is only available for some questions, if the admin sets that up

Comment (MartinGbz) [reply projects/safe/src/lib/style/survey.scss]: because this method worked I forgot that .content-left was a problem

**Pull request merged**